console/src/app/modules/policies/private-labeling-policy/private-labeling-policy.component.ts

```diff
@@ -238,17 +238,38 @@ export class PrivateLabelingPolicyComponent implements OnInit, OnDestroy {
         }
         break;
       case PolicyComponentServiceType.MGMT:
-        if (type === AssetType.LOGO) {
-          if (theme === Theme.DARK) {
-            return previewHandler(this.service.removeLabelPolicyLogoDark());
-          } else if (theme === Theme.LIGHT) {
-            return previewHandler(this.service.removeLabelPolicyLogo());
-          }
-        } else if (type === AssetType.ICON) {
-          if (theme === Theme.DARK) {
-            return previewHandler(this.service.removeLabelPolicyIconDark());
-          } else if (theme === Theme.LIGHT) {
-            return previewHandler(this.service.removeLabelPolicyIcon());
+        if ((this.previewData as LabelPolicy.AsObject).isDefault) {
+          const req0 = new AddCustomLabelPolicyRequest();
+          this.overwriteValues(req0);
+
+          return (this.service as ManagementService)
+            .addCustomLabelPolicy(req0)
+            .then(() => {
+              if (this.previewData) {
+                this.previewData.isDefault = false;
+              }
+              this.toast.showInfo('POLICY.TOAST.SET', true);
+
+              setTimeout(() => {
+                this.fetchData();
+              }, 1000);
```

Comment:
And why after a delayed second anyway? So the toast don't interfere with each other? :joy:

Reply:
eventual consistency.. it reloads after a new custom policy has been created

Reply:
had problems leaving it without timeout

Reply:
And why do we need it in the first place?

Reply:
because the logo is initially shown because it inherits its config from the instance.. first we have to create a custompolicy to be able to add a logo..

Reply:
so deletion of the logo is actually adding a custom policy with no logo